src/components/Blogs/Blogs.tsx

```diff
@@ -0,0 +1,127 @@
+import { Posts } from "blogs"
+import BlogPost from "components/BlogLanding/BlogList"
+import { Button } from "components/Button"
+import usePosts from "hooks/blogs"
+import React from "react"
+import styled from "styled-components"
+
+const BlogPage = styled.div`
+`
+
+const RecentPostsWrapper = styled.div`
+    margin-bottom: 40px;
+`
+
+const RecentPostDiv = styled.div`
+    padding:40px;
+    width: 100%;
+    margin: 0 auto;
+    display: inline-block;
+`
+
+
+const RecentImageDiv = styled.div`
+    width: 50%;
+    height: 300px;
+    float: left;
+    > .img {
+        width: 100%;
+        height: 100%;
+        border-radius: 10px;
+        box-shadow: 0px 0px 8px 0px #ada8a8;
+    }
+`
+
+const RecentPostText = styled.div`
+    padding: 20px;
+    float: right;
+    width: 40%;
+    > .PostTitle-Blog {
+        font-size: 18px;
+        font-weight: bold;
+        font-style: italic;
+        margin-bottom: 5px;
+    }
+    > .PostAuthor-Blog{
+        font-size: 12px;
+        font-weight: bold;
+        margin: 20px 0;
+    }
+    > .PostDescription-Blog {
+        margin: 10px 0;
+        font-size: 14px;
+    }
+    > .PostReadButton-Blog {
+        margin: 25px 0;
+        > .ButtonDiv {
+            padding: 10px 40px;
+            > .ButtonText {
+                font-size: 18px;
+            }
+        }
+    }
+`
+
+const Blogs: React.FC = () => {
+    const posts = usePosts()
+    const hasPosts = posts?.posts?.length > 0
+
+    if (!hasPosts) {
+        return (
+            <BlogPage>
+                <div>Stay tuned for upcoming blogs!</div>
```

Comment:
this isn't being displayed when there are 0 public blogs. Possible wrong logic on line 67, where we should be checking that there are no "public" blogs rather than just the number of blogs